### ISSUE TITLE
Manual Fan control

### ISSUE THREAD

**Issue by proneon267** [seerge/g-helper#291]
Hello
Thanks for writing this software. That bloated armoury crate seems to have dementia or something. It will literally forget to turn on the dGPU and sometimes even turn on the fan at high temperatures.

**Is your feature request related to a problem? Please describe.**
No matter the fan curve settings, the fans only run at or below 60% of their maximum rpm speed.

**Describe the solution you'd like**
A slider for controlling the fan speed rpm or something related. A more decoupled fan speed control that only increases the fan speed without boosting the CPU.

**Describe alternatives you've considered**
Make peace with the fans running only at or below 60% of their full speed.

**Additional context**
Ran Debloat script.
Model:  ASUS TUF Gaming A15 FA506QM_FA506QM (Ryzen 9 + RTX 3060)

**Current Settings:**
<img src="https://user-images.githubusercontent.com/45512885/234598472-5ad21964-9a6c-46c3-929f-5b50d25d5589.png" width=50%><img src="https://user-images.githubusercontent.com/45512885/234599405-d5be48bc-ba21-443d-93f6-32e6c3a6e169.png" width=50%><img src="https://user-images.githubusercontent.com/45512885/234599695-bcc8a38c-381f-4437-ac43-887ebf42f5f3.png" width=50%><img src="https://user-images.githubusercontent.com/45512885/234599695-bcc8a38c-381f-4437-ac43-887ebf42f5f3.png" width=50%>

**Comment by seerge** [comment]
@proneon267 What does it show in RPM (when you click on fan % label) ?

**Comment by proneon267** [comment]
@seerge At around 60% it shows 3000-3300 rpm.

**Comment by seerge** [comment]
1. Can you upload a log (as was asked in a issue template)
2. What RPM do you get if you set it to max in armoury ?

**Comment by proneon267** [comment]
Sorry, I thought I had attached it:
[log.txt](https://github.com/seerge/g-helper/files/11334563/log.txt)
![image](https://user-images.githubusercontent.com/45512885/234623895-8b08bc7e-c0b7-4a57-be31-c45b4ae0f292.png)

**Comment by seerge** [comment]
@proneon267 but in armoury your fans also are limited to same amount? looks like this is the max for your laptop ? :)

**Comment by seerge** [comment]
@proneon267 in a log I see that bios is not acceptinf fan curves at all:

```
4/26/2023 8:19:03 AM: FanCPU = 14-15-14-15-1F-27-37-62-5B-60-63-63-63-63-63-63 : 0
4/26/2023 8:19:03 AM: FanGPU = 14-15-44-48-4C-50-54-62-60-63-63-63-63-63-63-63 : 0
4/26/2023 8:19:03 AM: ASUS BIOS rejected fan curve, resetting mode to 1
```

Does custom mode in armoury allow you to customize fans?

**Comment by proneon267** [comment]
@seerge There is no custom mode for TUF notebooks, only ROG notebooks have manual mode that has fan curve controls.

**Comment by seerge** [comment]
@proneon267 then probably your laptop doesn't support custom fan curve at all :) if even armoury doesn't have that option.

Some TUF laptops do have fan curves tho

**Comment by proneon267** [comment]
@seerge Yes only TUF notebooks after 2021 have fan curve controls. Weirdly enough, on linux, fan controls are working normally. I'll check if lm_sensors works and let you know.

Anyways, thanks for helping :)

**Comment by seerge** [comment]
@proneon267 well, for now i'm aware only about only one way to set fan curves (same as armoury does) and use it, and from log it's clear that bios responds 0 - so it doesn't accept that :)
